### ISSUE TITLE
Couldn't run containers

### ISSUE THREAD

**Issue by florianhidayat** [aws/amazon-ecs-cli#551]
### Summary
I've been having this trouble for a few days. Basically, whenever I try to run compose up, I says it couldn't run the containers (reason="RESOURCE:CPU"). It seems like it cannot find the CloudFormation stack (i.e. EC2ContainerService-chorus-2), even though I've included that inside my config file.

### Description
`ecs-cli compose --file docker-compose.prod.yml --debug up`

ecs-cli v1.6.0
go version go1.9.2 darwin/amd64
I'm using a Mac.

### Config files

* docker-compose.yml
```
version: '3'

services:
  db:
    image: mongo:3.4.3
    command: mongod --smallfiles --quiet
    environment:
      - XXX_XXXX=XXXXX
    ports:
      - "XXXXX:27017"
    volumes:
      - /home/data:/data
      - /home/data/db:/data/db
      - /home/data/configdb:/data/configdb
      - /home/mongo-entrypoint:/docker-entrypoint-initdb.d

  redis:
    image: redis:latest
    ports:
      - "6379:6379"
    command: --requirepass XXXXXXXX

  app:
    image: xxxxxx/xxxxxx_backend:latest
    links:
      - "db"
      - "redis"
    environment:
      - XXXXX=XXXXXX
      …
      …
      …
    ports:
      - "80:3000"
    volumes:
      - /home/logs/app:/myapp/log/
    command: ./scripts/start.sh
```

* ecs-params.yml
```
version: 1
task_definition:
  services:
    db:
      cpu_shares: 128
      mem_limit: 480MB
    redis:
      cpu_shares: 128
      mem_limit: 128MB
    app:
      cpu_shares: 768
      mem_limit: 1350MB
```
* ~/.ecs/config
```
version: v1
default: default
clusters:
  default:
    cluster: chorus-2
    region: us-west-2
    cfn-stack-name: EC2ContainerService-chorus-2
    default_launch_type: EC2
```

### Observed Behavior
I receive the following:
> DEBU[0000] Parsing the compose yaml...                  
> DEBU[0000] Docker Compose version found: 2              
> DEBU[0000] Parsing v1/2 project...                      
> DEBU[0000] Opening compose files: docker-compose.prod.yml 
> DEBU[0000] [0/1] [app]: Adding                          
> DEBU[0000] [0/1] [db]: Adding                           
> DEBU[0000] [0/3] [redis]: Adding                        
> DEBU[0000] [0/3] [default]: EventType: 32               
> DEBU[0000] [0/3] [default]: EventType: 32               
> DEBU[0000] [0/3] [default]: EventType: 32               
> DEBU[0000] Parsing the ecs-params yaml...               
> DEBU[0000] Transforming yaml to task definition...      
> INFO[0000] Using ecs-params value as override (was 128 but is now 768)  option name=MemoryReservation service name=app
> INFO[0000] Using ecs-params value as override (was 1350 but is now 1350)  option name=MemoryReservation service name=app
> INFO[0000] Using ecs-params value as override (was 128 but is now 128)  option name=MemoryReservation service name=db
> INFO[0000] Using ecs-params value as override (was 450 but is now 480)  option name=MemoryReservation service name=db
> INFO[0000] Using ecs-params value as override (was 128 but is now 128)  option name=MemoryReservation service name=redis
> INFO[0000] Using ecs-params value as override (was 122 but is now 128)  option name=MemoryReservation service name=redis
> DEBU[0002] cache miss                                    taskDef="{\n\n}" taskDefHash=f7fefa7056cd940a6dc5238134eecde7
> INFO[0002] Using ECS task definition                     TaskDefinition="chorus_backend:43"
> INFO[0002] Couldn't run containers                       reason="RESOURCE:CPU"

**Comment by PettitWesley** [comment]
@florianhidayat the error message you're getting indicates that ECS could not find a container instance in your cluster that has the required amount of CPU available (768 + 128 + 128 = 1024). I'd recommend going into the ECS Web Console and checking the usage statistics for your container instances in cluster `chorus-2`.

I do not believe this is related to the CloudFormation template, the ECS CLI only uses the cluster name for the compose command- the CloudFormation template name is only used when creating, scaling, or deleting a cluster.

Let me know if you need more help!

**Comment by florianhidayat** [comment]
@PettitWesley I figured it wasn't worth the extra hassle so we decided to just create a new cluster with the same settings. This way our new CloudFormation will have a name with a format that ECS respects.

Regarding the issue, I'm absolutely sure it wasn't that ECS couldn't find the instance with the enough CPU units, it simply couldn't find any container instance. Whatever amount of units I put inside my compose file, it just wouldn't work. And by the way, we initially created the cluster with CLI ver. 0.6.6.

My guess it that cfn-stack-name doesn't really do anything, or perhaps there's something wrong with how it's implemented (or maybe I was wrong in how I used it, but I don't think so as it seems pretty straightforward). Maybe it can be taken a look at.
